tools/modules/update.sh

```diff
@@ -22,21 +22,24 @@
 # repositories are at the correct version and will configure and build them.
 
 wget_copy_archive() {
+  # The dependency to download (llvm, clang).
```

Comment:
e.g., "This must be a repository hosted by the LLVM mirror" or words to that effect.

Reply:
:+1: